jdaviz/configs/cubeviz/plugins/mixins.py

```diff
@@ -14,7 +15,7 @@ def slice_component_label(self):
 
     @property
     def slice_display_unit_name(self):
-        return 'spectral'
+        return 'spectral' if self.default_class != NDDataArray else 'temporal'
```

Comment:
this logic feels a little fragile, would it be better to override in a subclass (both here and in the slice plugin)?

Reply:
I agree that it's fragile. I hesitate to make a subclass to override only this.

Reply:
Sounds good!